assets/hip-1056/protobuf/stream/block_proof.proto

```diff
@@ -0,0 +1,212 @@
+     * the Block Stream correctly.
+     * Stateless (non-state-processing) clients MUST use this value to
+     * construct the block merkle tree.
+     */
+    bytes start_of_block_state_root_hash = 3;
+
+    /**
+     * A TSS signature for one block.<br/>
+     * This is a single signature representing the collection of partial
+     * signatures from nodes holding strictly greater than 2/3 of the
+     * current network "weight" in aggregate. The signature is produced by
+     * cryptographic "aggregation" of the partial signatures to produce a
+     * single signature that can be verified with the network public key,
+     * but could not be produced by fewer nodes than required to meet the
+     * threshold for network stake "weight".
+     * <p>
+     * This message MUST make use of a threshold signature scheme like `BLS`
+     * which provides the necessary cryptographic guarantees.<br/>
+     * This signature SHALL use a TSS signature to provide a single signature
+     * that represents the consensus signature of consensus nodes.<br/>
+     * The exact subset of nodes that signed SHALL neither be known nor
+     * tracked, but it SHALL be cryptographically verifiable that the
+     * threshold was met if the signature itself can be validated with
+     * the network public key (a.k.a `LedgerID`).
+     */
+    bytes block_signature = 4;
+
+    /**
+     * A set of hash values along with ordering information.<br/>
+     * This list of hash values form the set of sibling hash values needed to
+     * correctly reconstruct the parent hash, and all hash values "above" that
+     * hash in the merkle tree.
+     * <p>
+     * A Block proof can be constructed by combining the sibling hashes for
+     * a previous block hash and sibling hashes for each entry "above" that
+     * node in the merkle tree of a block proof that incorporates that previous
+     * block hash. This form of block proof may be used to prove a chain of
+     * blocks when one or more older blocks is missing the original block
+     * proof that signed the block's merkle root directly.
+     * <p>
+     * This list MUST be ordered from the sibling of the node that contains
+     * this block's root node hash, and continues up the merkle tree to the
+     * root hash of the signed block proof.
+     * <p>
+     * If this block proof has a "direct" signature, then this list MUST be
+     * empty.<br/>
+     * If this list is not empty, then this block proof MUST be verified by
+     * first constructing the "block" merkle tree and computing the root hash
+     * of that tree, then combining that hash with the values in this list,
+     * paying attention to the first/second sibling ordering, until the root
+     * merkle hash is produced from the last pair of sibling hashes. That
+     * "secondary" root hash MUST then be verified using
+     * the value of `block_signature`.
+     */
+    repeated MerkleSiblingHash sibling_hashes = 5;
```

Comment:
There is one more field added here on main @Nana-EC 
```
oneof verification_reference {
        /**
         * The id of the hinTS scheme this signature verifies under.
         */
        uint64 scheme_id = 6;

        /**
         * The explicit hinTS key this signature verifies under; useful when
         * the verifier can easily check whether a chain-of-trust proof
         * exists for a key, but does does not have any context of the
         * latest hinTS schemes published in the block stream.
         */
        bytes verification_key = 7;
    }
```

Reply:
Added